scripts/firedrake-install

```diff
@@ -619,10 +619,11 @@ elif options.get("with_blas") is not None:
         petsc_options.add("--CFLAGS=-I{}/include".format(options["with_blas"]))
 elif osname == "Darwin":
     brew_prefix = check_output(['brew', '--prefix'])[:-1]
+    brew_gcc_prefix = brew_prefix+"/opt/gcc/lib/gcc/11"
```

Comment:
Should the 11 here really be hard-coded? What happens when that changes?

Reply:
That's a great point -- we can parse the output from `brew info --json gcc` or does that seem too complicated?

Reply:
Alright it turned out to be super easy, this is ready for another look.